app/controllers/application_letters_controller.rb

```diff
@@ -57,6 +57,6 @@ def set_application
 
     # Only allow a trusted parameter "white list" through.
     def application_params
-      params.require(:application_letter).permit(:motivation, :event_id, :status)
+      params.require(:application_letter).permit(:motivation, :user_id, :event_id, :status)
```

Comment:
@cmfcmf are there already tests for this? if yes, we also broke the tests.
If no, maybe your group should add some, since this seems like an important security feature :)